vita3k/interface.cpp

```diff
@@ -267,7 +270,7 @@ static ExitCode load_app_impl(Ptr<const void> &entry_point, HostState &host, con
         return InvalidApplicationPath;
 
     vfs::FileBuffer params;
-    bool params_found = vfs::read_app_file(params, host.pref_path, host.io.title_id, "sce_sys/param.sfo");
+    bool params_found = vfs::read_app_file(params, host.pref_path, host.app_title_id, "sce_sys/param.sfo");
```

Comment:
Remove this, get broken load app

Reply:
You can see that people are actually confused by this.

Reply:
We should really move this to gui state.

Reply:
No relate, is all time writed like that is not news get by my code